src/template.rs

```diff
@@ -143,10 +144,11 @@ pub struct HelperTemplate {
     pub inverse: Option<Template>,
     pub block: bool,
     pub chain: bool,
+    pub indent_before_write: bool,
```

Comment:
I think we can make this field private, as well as the one for `DecoratorTemplate`

Reply:
Sure!  I need them in `render_helper`, so I guess `pub(crate)` it is.